docs/sphinx/source/index.rst

```diff
@@ -38,7 +38,7 @@ Please see our
 `PVSC 2014 paper <http://energy.sandia.gov/wp/wp-content/gallery/uploads/PV_LIB_Python_final_SAND2014-18444C.pdf>`_
 and
 `PVSC 2015 abstract <https://github.com/UARENForecasting/pvlib-pvsc2015/blob/master/pvlib_pvsc_42.pdf?raw=true>`_ 
-for more information.
+(and the `final paper <https://github.com/pvlib/pvsc2015>`_) for more information.
```

Comment:
"final paper" doesn't seem like the right way to describe this. A better way might be to change "PVSC 2015 abstract" to "PVSC 2015 paper" with the correct pdf link, then replace "and the final paper" with something like  "GitHub repository". I don't see any benefit to keeping the link to the abstract.